apps/condo/domains/billing/components/BillingPageContent/MainContent.tsx

```diff
@@ -59,7 +59,7 @@ export const MainContent: React.FC<MainContentProps> = ({
             result.push({
                 label: extensionPageTitle,
                 key: EXTENSION_TAB_KEY,
-                children: <IFrame src={appUrl} reloadScope='organization' withPrefetch withLoader withResize/>,
+                children: <IFrame src={appUrl} reloadScope='organization' withPrefetch withLoader withResize scrolling='yes' />,
```

Comment:
it's better to control it by child window. @SavelevMatthew check it please

Reply:
Why do you added such prop? 

We use bridge to control this behaviour (iframe height). Iframe should not be scrollable

Reply:
i initially wanted to scroll the iFrame window so that the ActionBar at the bottom of it would work. but it looks bad since the parent window also scrolls and there are two scrollbars.

i'll remove this prop, but i'm still unsure how to solve the problem of ActionBar not floating